DFE-Digital/claim-additional-payments-for-teaching-qts-api
# [CAPT-309] Implement review apps

## What
Create new infrastructure to receive review apps created in https://github.com/DFE-Digital/claim-additional-payments-for-teaching

## How to review
- Run `make review terraform-plan`
- Check https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2043
